libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -331,6 +333,10 @@ bool GLSLPostProcessor::process(const std::string& inputShader, Config const& co
     const char* shaderCString = inputShader.c_str();
     tShader.setStrings(&shaderCString, 1);
 
+    // This allows shaders to query if they will be run through glslang.
+    // OpenGL shaders without optimization, for example, won't have this define.
+    tShader.setPreamble("#define FILAMENT_GLSLANG 1\n");
```

Comment:
It should probable not be defined to a value, since it should be tested with #ifdef / #ifndef